resources/l10n/pl/LC_MESSAGES/loot.po

```diff
@@ -1,1072 +1,1232 @@
+msgid "&Game"
+msgstr "&Gra"
 
-#: gui/editor.cpp:493
-msgid "Save Changes"
-msgstr "Zapisz zmiany"
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:668
+msgid "&Edit Groups..."
+msgstr "&Edytuj grupy..."
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:670
+msgid "Searc&h Cards..."
+msgstr "Szukaj ka&rty..."
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:672
+msgid "Copy &Load Order"
+msgstr "Kopiuj &załaduj kolejność"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:674
+msgid "&Copy Content"
+msgstr "&Kopiuj Zawartość"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:676
+msgid "&Refresh Content"
+msgstr "&Odśwież zawartość"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:678
+msgid "Redate &Plugins..."
+msgstr "Przedatuj &Wtyczki..."
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:680
+msgid "&Fix Ambiguous Load Order"
+msgstr "&Napraw niejednoznaczną kolejność ładowania"
+
+#. translators: This string is an action in the Game menu.
+#: src/gui/qt/main_window.cpp:682
+msgid "Clear All &User Metadata..."
+msgstr "Usuń wszystkie &meta informacje użytkownika..."
+
+#. translators: The mnemonic in this string shouldn't conflict with other
+#. menus or sidebar sections.
+#: src/gui/qt/main_window.cpp:686
+msgid "&Plugin"
+msgstr "&Wtyczka"
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:688
+msgid "Copy &Plugin Name"
+msgstr "Kopiuj &nazwe wtyczki"
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:690
+msgid "Copy &Card Content"
+msgstr "Kopiuj &zawartość karty"
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:692
+msgid "Copy &Metadata"
+msgstr "Kopiuj &metadane"
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:694
+msgid "&Edit Metadata..."
+msgstr "&Edytuj Metadane..."
+
+#. translators: This string is an action in the Plugin menu.
+#: src/gui/qt/main_window.cpp:696
+msgid "Clear &User Metadata..."
+msgstr "Wyczyść metadane &użytkownika..."
+
+#. translators: The mnemonic in this string shouldn't conflict with other
+#. menus or sidebar sections.
+#: src/gui/qt/main_window.cpp:700
+msgid "&Help"
+msgstr "Po&moc"
+
+#. translators: This string is an action in the Help menu.
+#: src/gui/qt/main_window.cpp:702
+msgid "&View Documentation"
+msgstr "&Zobacz dokumentacje"
+
+#. translators: This string is an action in the Help menu.
+#: src/gui/qt/main_window.cpp:704
+msgid "&Join Discord Server"
+msgstr "&Dołącz na server discorda"
+
+#. translators: This string is an action in the Help menu.
+#: src/gui/qt/main_window.cpp:706
+msgid "&About"
+msgstr "&O"
```

Comment:
Is this missing the rest of the word?

Reply:
I wouldn't think so, apparently that's a valid translation of `About`.

Reply:
OK